sub-packages/bionemo-geneformer/src/bionemo/geneformer/run/main.py

```diff
@@ -28,70 +28,71 @@
 from bionemo.llm.train import NsysConfig, train
 
 
-def main():  # noqa: D103
-    def parse_args():
-        parser = argparse.ArgumentParser(description="Run Geneformer pretraining")
-        parser.add_argument("--config", type=str, required=True, help="Path to the JSON configuration file")
-        parser.add_argument(
-            "--model-config-cls",
-            default=ExposedGeneformerPretrainConfig,
-            required=False,
-            help="fully resolvable python import path to the ModelConfig class. Builtin options are ExposedGeneformerPretrainConfig and ExposedFineTuneSeqLenBioBertConfig.",
-        )
-        parser.add_argument(
-            "--data-config-cls",
-            default=GeneformerPretrainingDataConfig,
-            required=False,
-            help="fully resolvable python import path to the class.",
-        )
-        parser.add_argument(
-            "--resume-if-exists",
-            default=False,
-            action="store_true",
-            help="Resume training if a checkpoint exists that matches the current experiment configuration.",
-        )
+def parse_args():  # noqa: D103
+    parser = argparse.ArgumentParser(description="Run Geneformer pretraining")
+    parser.add_argument("--config", type=str, required=True, help="Path to the JSON configuration file")
+    parser.add_argument(
+        "--model-config-cls",
+        default=ExposedGeneformerPretrainConfig,
+        required=False,
+        help="fully resolvable python import path to the ModelConfig class. Builtin options are ExposedGeneformerPretrainConfig and ExposedFineTuneSeqLenBioBertConfig.",
+    )
+    parser.add_argument(
+        "--data-config-cls",
+        default=GeneformerPretrainingDataConfig,
+        required=False,
+        help="fully resolvable python import path to the class.",
+    )
```

Comment:
I think it would be less confusing to make the default match the type of what a user would enter, like a string. As is I don't see an example of how to change this to something else.

Reply:
`bionemo.geneformer.run.config_models.ExposedFineTuneSeqLenBioBertConfig` for example.